plugins/module_utils/dnac.py

```diff
@@ -386,15 +400,13 @@ def camel_to_snake_case(self, config):
             return config
         return new_config
 
-
-def log(msg, frameIncrement=0):
+def log(msg,  level='info', frameIncrement=0):
     with open('dnac.log', 'a') as of:
         callerframerecord = inspect.stack()[1 + frameIncrement]
         frame = callerframerecord[0]
         info = inspect.getframeinfo(frame)
         d = datetime.datetime.now().replace(microsecond=0).isoformat()
-        of.write("---- %s ---- %s@%s ---- %s \n" % (d, info.lineno, info.function, msg))
-
+        of.write("---- %s ---- %s@%s ---- %s: %s \n" % (d, info.lineno, info.function, level.upper(), msg))
```

Comment:
Can we print any sample log messages in the PR description?

Reply:
done!